dbfit-java/core/src/main/java/dbfit/util/FitNesseTestHost.java

```diff
@@ -9,6 +10,7 @@
 
     static {
         TypeAdapter.registerParseDelegate(BigDecimal.class, BigDecimalParseDelegate.class);
+        TypeAdapter.registerParseDelegate(BigInteger.class, BigIntegerParseDelegate.class);
```

Comment:
Is the type in MySQL 64 but unsigned and so will this class support the same range of values?